src/funding/applepay/config.jsx

```diff
@@ -22,11 +22,18 @@ export function getApplePayConfig(): FundingSourceConfig {
         applepay: true,
       };
     },
-    eligible: ({ components }) => {
+    eligible: ({ components, enableFunding }) => {
       /** If the JS SDK Script Includes Standalone ApplePay Component the Exclude the ApplePay Button From the Vertical Stack
        * https://paypal.atlassian.net/browse/DTALTPAY-1232
+       * Only render applepay in vertical stack if applepay is present in enable-funding list
+       * https://paypal.atlassian.net/browse/DTALTPAY-1236
        */
-      return !components?.includes(FUNDING.APPLEPAY);
+      const isEnableFundingApplepay =
+        enableFunding && enableFunding.indexOf(FUNDING.APPLEPAY) !== -1;
+      return (
+        !components?.includes(FUNDING.APPLEPAY) &&
+        Boolean(isEnableFundingApplepay)
+      );
```

Comment:
```suggestion
      const isEnableFundingApplepay = Boolean(
        enableFunding?.includes(FUNDING.APPLEPAY)
      );
      return !components?.includes(FUNDING.APPLEPAY) && isEnableFundingApplepay;
```

Nitpick: I would use `.includes` here instead of `.indexOf`, for simplicity and to match the existing code on line 29.  

PR is looking good!  🥳

Reply:
Thank you. Btw - The linter / flow were triggering for `isEnableFundingApplepay` without it converting to Boolean